sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb

```diff
@@ -505,9 +505,7 @@
     "    tags={\"my_custom_tag\": \"My custom value\"},\n",
     ")\n",
     "\n",
-    "image_classification_job.set_limits(\n",
-    "    timeout_minutes=60, max_trials=5, max_concurrent_trials=2\n",
-    ")"
+    "image_classification_job.set_limits(max_trials=5, max_concurrent_trials=2)"
```

Comment:
Can we please move all max_trials to 10 for automode. There is a helper script running which searches for max_trials=10 and modifies it to 2 runs for tests/github workflows... We can't search for all max_trials because this is a shared arg between all verticals...

```suggestion
    "image_classification_job.set_limits(\n",
    "    max_trials=10,\n",
    "    max_concurrent_trials=2,\n",
    ")\n",
```

Reply:
Done. The script behavior sounds risky though - why wouldn't we just set max_trials to 2 directly in our examples? Also, is the behavior documented somewhere?